composer.json

```diff
@@ -39,6 +39,7 @@
       "drupal/publishcontent": "^1.3",
       "drupal/quicklink": "^1.3",
       "drupal/quick_node_clone": "^1.14",
+      "drupal/rabbit_hole": "^1.0@beta",
```

Comment:
beta... bummer...

Reply:
Yeah, I wish it weren't beta. I'm actually testing this one out for specific access controls and what to do when you go to a page you don't have access to - apparently you can specify redirect or custom messages, etc. So, that'll (hopefully) work for things like a member-only page and we want to redirect to the oAuth server first if you aren't logged in...